From e781a53dcae5c90a4d258c7e2e1082a52b5765ae Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Mon, 10 Jun 2024 13:18:33 -0700
Subject: [PATCH 4/7] missing reference to strlen

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 filament/include/filament/MaterialInstance.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/filament/include/filament/MaterialInstance.h b/filament/include/filament/MaterialInstance.h
index 0c6ef2057..91fce4992 100644
--- a/filament/include/filament/MaterialInstance.h
+++ b/filament/include/filament/MaterialInstance.h
@@ -28,6 +28,8 @@
 
 #include <math/mathfwd.h>
 
+#include <cstring>
+
 namespace filament {
 
 class Material;
-- 
2.45.1

